config/locales/controllers/studentship/course_enrollments.en.yml

```diff
@@ -30,10 +31,11 @@ en:
       errors:
         already_enrolled_at_group: The maximum amount of ECTS that can be selected from the course group is selected.
         empty_selected_courses_list: The list of selected courses is empty.
-        quota_full: The course has reached its quota.
```

Comment:
burada ortaya masumane bir proposal atıyorum. "course quota" mı, "course capacity" mi? bence ikincisi. aklıma direkt olarak odtü'lülerin belası olan uyarı geliyor. "Course capacity is full!"

Reply:
Bunu bilahare tartışalım :-)